server/queue/queue-memory/pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>james-server</artifactId>
+        <groupId>org.apache.james</groupId>
+        <version>3.1.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>james-server-queue-memory</artifactId>
+
+    <name>Apache James :: Server :: Mail Queue :: Memory</name>
+
+    <dependencies>
+        <dependency>
```

Comment:
did you verified everything here is required ? I would think queue-api to be not far from enough.

Reply:
no answer to that ?

Reply:
I cleaned it!